### DISCUSSION TITLE
CALL Get Api RowId with two primary Key

### DISCUSSION THREAD

**Discussion by syji35** [nocodb/nocodb#3726] [Q&A]
I have a table in postgre database with a composite primary key (Column P1 and P2)

How to call the API specifying the two primary keys ?
I try this : 
`
GET 'https://nocode.csm.ovh/api/v1/db/data/v1/DATA/Data/views/Viewer/valueP1__ValueP2' 
`
I have no results.

Thanks

**Comment by o1lab** [comment]
Three underscores between p1 and p2

**Reply by syji35** [reply]
Same result :(

**Reply by o1lab** [reply]
are you sure you've two primary keys ? :)

**Reply by syji35** [reply]
As you can see : 
Schema :  
![image](https://user-images.githubusercontent.com/45880189/191470570-67615554-d1cf-4269-a2ba-4b2b0184029a.png)

Example data :   
![image](https://user-images.githubusercontent.com/45880189/191471239-0725355f-a022-4b5a-b6f1-81f775684269.png)

Example request swagger :   
![image](https://user-images.githubusercontent.com/45880189/191471600-dda43993-cc45-47a0-8102-3a74e0692145.png)

**Reply by syji35** [reply]
any idea?
